safe_authenticator/src/app_auth.rs

```diff
@@ -188,7 +218,7 @@ pub fn authenticate(client: &AuthClient, auth_req: AuthReq) -> Box<AuthFuture<Au
                 }
             }
         })
-        .and_then(move |(app, app_state, app_id)| {
+        .and_then(move |(app, app_state, app_id, token)| {
             match app_state {
                 AppState::Authenticated => {
                     // Return info of the already registered app
```

Comment:
Can you update the comments here and the func description too. With these changes those no longer make sense 👍